README.md

```diff
@@ -27,6 +27,11 @@ Then you can start the session as usual
 
 Then the `$_SESSION` array is accessible.
 
+You can configure rules for managing conflicts. Just add element to the $conflictRules[regex] = rule. The possible rules are
```

Comment:
Not really easy to understand. Maybe adding a sample would be useful?

Also, beware of Markdown syntax for lists.